pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -112,6 +112,12 @@ func (s *Service) reconcileV2LB(lbSpec *infrav1.AWSLoadBalancerSpec) error {
 	// set up the type for later processing
 	lb.LoadBalancerType = lbSpec.LoadBalancerType
 	if lb.IsManaged(s.scope.Name()) {
+		// Reconcile the target groups and listeners from the spec and the ones currently attached to the load balancer.
+		_, _, err := s.reconcileTargetGroupsAndListeners(lb, lbSpec)
```

Comment:
```suggestion
		_, _, err := s.reconcileTargetGroupsAndListeners(spec, lbSpec)
```
That's why ELBListeners is empty!

Reply:
Thanks for digging into this! Appreciate the help.

I'll review these variables, since we have 3 representations of roughly the same thing. `spec`, `lbSpec`, and `lb` probably need to have clearer names.